elevation_mapping_demos/scripts/tf_to_pose_publisher.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+import rospy
+import geometry_msgs.msg
+import std_msgs
+import tf
+
+def callback(newPose):
+    global publisher, tfListener, from_frame, to_frame
```

Comment:
Docstring for callback? [[Source](https://realpython.com/python-pep8/)]

Reply:
I added docstring to the function and more comments to the code.

Reply:
Solid :+1: